fjord/alerts/migrations/0003_auto_20150225_1213.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
you can rename the migration files to something more descriptive if you want. It would make browsing the migrations folder easier but that's kind of a rare use case.